pkg/controllers/deprovisioning/orchestration/metrics.go

```diff
@@ -0,0 +1,51 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package orchestration
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/aws/karpenter-core/pkg/metrics"
+)
+
+func init() {
+	crmetrics.Registry.MustRegister(deprovisioningReplacementNodeInitializedHistogram, deprovisioningReplacementNodeLaunchFailedCounter)
+}
+
+const (
+	deprovisioningSubsystem = "deprovisioning"
+	deprovisionerLabel      = "deprovisioner"
+)
+
+var (
+	deprovisioningReplacementNodeInitializedHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: deprovisioningSubsystem,
+			Name:      "replacement_machine_initialized_seconds",
```

Comment:
Do we want a new metric for this? Can we reuse our existing metric and add a "replacement" label?

Reply:
There's another comment that @jonathan-innis about this. I want to keep this here as it was part of the initial code, and then re-work it into another controller where it's better lived in a separate PR.